packages/apps-v5/src/commands/ps/index.js

```diff
@@ -68,10 +68,10 @@ async function printAccountQuota(context, heroku, app, account) {
   let appHours = Math.floor(appQuotaUsed / 60)
   let appMinutes = Math.floor(appQuotaUsed % 60)
 
-  cli.log(`Free dyno hours quota remaining this month: ${hours}h ${minutes}m (${percentage}%)`)
-  cli.log(`Free dyno usage for this app: ${appHours}h ${appMinutes}m (${appPercentage}%)`)
-  cli.log('For more information on dyno sleeping and how to upgrade, see:')
-  cli.log('https://devcenter.heroku.com/articles/dyno-sleeping')
+  cli.log(`Eco dyno hours quota remaining this month: ${hours}h ${minutes}m (${percentage}%)`)
+  cli.log(`Eco dyno usage for this app: ${appHours}h ${appMinutes}m (${appPercentage}%)`)
+  cli.log('For more information on Eco dyno hours, see:')
+  cli.log('https://devcenter.heroku.com/articles/eco-dyno-hours')
```

Comment:
Same as above -- this should still reflect free dyno hours if not subscribed to eco while free is still available.

Reply:
👍